libiocage/lib/Jails.py

```diff
@@ -26,122 +28,84 @@ def __init__(self,
         libiocage.lib.helpers.init_host(self, host)
         self.zfs = libzfs.ZFS(history=True, history_prefix="<iocage>")
 
-    def list(self, filters=None):
-
-        if len(filters) == 1:
-            chars = "+*="
-            name = filters[0]
-            if not any(x in name for x in chars):
-                single_jail = libiocage.lib.Jail.Jail(
-                    {
-                        "name": name
-                    },
-                    logger=self.logger,
-                    host=self.host,
-                    zfs=self.zfs
-                )
-                return [single_jail]
-
-        jails = self._get_existing_jails()
-
-        if filters is not None:
-            return self._filter_jails(jails, filters)
-        else:
-            return jails
+        self._filters = None
+        self.filters = filters
+        list.__init__(self, [])
```

Comment:
Debug?

Reply:
Nope. Line 31 makes the `_filters` property available, the line below calls the `filters` property setter.